pkgpanda/__init__.py

```diff
@@ -43,13 +43,25 @@
 username_regex = "^dcos_[a-z0-9_]+$"
 
 
+def _is_root():
+    return os.geteuid() == 0
+
+
+def _set_sysctl(sysctl_value, sysctl_var):
+    try:
+        sysctl_setting = "{key}={value}".format(key=sysctl_var, value=sysctl_value)
+        check_call(["/sbin/sysctl", "-w", sysctl_setting])
+    except CalledProcessError as _:
```

Comment:
also: When not going to use the value, shouldn't bother catching it with a name. Just `except CalledProcessError:` would work